shapely/algorithms/polylabel.py

```diff
@@ -29,8 +29,9 @@ def polylabel(polygon, tolerance=1.0):
     shapely.errors.TopologicalError
         If the input polygon is not a valid geometry.
 
-    Example
-    -------
+    Examples
+    --------
+    >>> from shapely.algorithms.polylabel import polylabel
```

Comment:
```suggestion
    >>> from shapely.ops import polylabel
```

In the manual.rst we use this import